### PR TITLE
Added demo description

### PR DESCRIPTION
**Pull request by BaardBouvet**

### CODE REVIEW

File: demo.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Public data sources
+-------------------
+
+The demo combines `company data from DBpedia <https://dbpedia.org/ontology/Company>`_, `legal entities from the Norwegian enhetsregisteret <https://en.wikipedia.org/wiki/Entity_Registry>`_ and more. Combining an English source and a Norwegian source shows how different languages play together in Sesam.
+
+Multi-tenancy
+-------------
+
+The demo also contains a setup for handling multiple tenants in one subscription. The setup contains the shared data from the public data sources, and a generator that uses a configuration template in `global-config` to generate a configuration for each tenant defined in `global-customer`. These configurations are added as separate :ref:`config groups <concepts-configgroup>`. The configuration template uses a copy of the shared data for each tenant to keep the data isolated between the tenants.
```
Comment (grove) [inline demo.rst:37]: I think the back-ticks need to be double.

File: demo.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Hubspot companies
+-----------------
+
+The demo implements a bi-directional synchronization of the `companies in Hubspot <https://developers.hubspot.com/docs/api/crm/companies>`_ data.
+
+The global model
+----------------
+
+The global model in the demo is a subset of the Hubspot model and contains mappings for the Hubspot properties that we want to manage with Sesam.
```
Comment (grove) [inline demo.rst:22]: OK, but what is a global model. I think the reader needs a more context.

File: demo.rst (hunk 1 above)
Comment (grove) [inline demo.rst:37]: What is a "configuration template"?

**Review by grove — APPROVED** [review]

**Review by JonasAls89 — APPROVED** [review]
lgtm 👍

**Pull request merged**